### PR TITLE
Feature/upgrade webext 3.2.0

### LINKED ISSUE #29: Upgrade to web-ext@3
https://github.com/mozilla/web-ext/releases/tag/3.2.0

Upgrading would also allow us to develop in Chrome too!

### PR DESCRIPTION
**Pull request by filoxo**
Resolves #29

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,22 +77,55 @@ If you would like to request a feature to be added, please open an issue with th
...
+
+   - This process is somewhat convoluted but needed in order to save preferences and any additional extensions
+   - Before starting any processes, open the Chrome Profiles directory
+     - Mac: `~/Library/Application Support/Google/Chrome`
+     - Windows: `%LOCALAPPDATA%\Google\Chrome\User Data`
+     - See the [Chromium User Data Directory docs](https://chromium.googlesource.com/chromium/src/+/master/docs/user_data_dir.md) for other platforms/chrome builds
+   - In that folder, take note of the Profile folders (eg. named "Profile 1", "Profile 2", etc. on Mac)
+   - Open Chrome and create a new profile
+   - Return to the Chrome user data folder and locate the newly created Profile folder
+   - Rename the folder to "single-spa-inpsector-dev" (for convenience)
```
Comment (filoxo) [inline README.md]: typo: `inpsector`

File: README.md [hunk 2]
```diff
@@ -77,22 +77,55 @@ If you would like to request a feature to be added, please open an issue with th
...
+#### Create a Firefox dev profile
 
 Currently, development happens by default in Firefox. If you would like Firefox to remember any settings that you change to Firefox itself, this project is configured to use a profile called "single-spa-inspector-dev". To create this profile, go to [about:profiles](about:profiles). Firefox will use that profile and remember any changes you make (e.g. devtools location, devtools dark mode, etc.)
 
-### Debugging
+#### Debugging
 
 Once single-spa Inspector is running, open a new tab and navigate to [about:debugging](about:debugging). single-spa Inspector should be listed as a Temporary Extension, and a "Debug" control should be displayed. Click on this to enable devtools for the extension. In the upper-right corner, click on the divided square icon next to the 3-dot menu, and select `/build/panel.html` as the target. You can now inspect the inspector UI as you would a normal webpage.
 
+## Chrome
```
Comment (frehner) [inline README.md]: nit: Chrome has `##` but firefox has `###`

File: README.md [hunk 3]
```diff
@@ -77,22 +77,55 @@ If you would like to request a feature to be added, please open an issue with th
...
+
+1. Install Chrome
+1. Create a Chrome profile
```
Comment (frehner) [inline README.md]: for consistency's sake do we want to make this a linkable section? or remove the section from FF?

File: README.md (hunk 2 above)
Comment (filoxo) [reply README.md]: Good catch. Chrome should also be `###`!

File: README.md (hunk 3 above)
Comment (filoxo) [reply README.md]: I think I'd prefer to remove it. I don't think installing a browser is a difficult thing to find, and our users likely already have these two installed on their machines.

Comment (filoxo) [reply README.md]: Oh wait, never mind the above comment. Let me see if I can find a resources from Chrome on how to do this.

Comment (filoxo) [reply README.md]: OK I put a link to it at the bullet point underneath this numbered point.

**Pull request merged**